mention fsproj files as an example of other dotnet reference files (#44031)
dotnet/docs: docs/core/tools/dotnet-add-reference.md

  dotnet add app/app.csproj reference lib/lib.csproj
  ```

> - Add a compatible .NET language project (e.g. F#) referenc, works in both ways:

Reviewer: ```suggestion
- Add a compatible .NET language (for example, F#) project reference, which works in both directions:
```
